src/01-functions.js

```diff
@@ -112,7 +137,14 @@ function averageThree() {}
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a, b) {
+  let x = a
+  let y = b
+  if (typeof (x) === typeof (y)) {return true;
+  }
+  else {return false;
+  }
```

Comment:
Could you do this without using the conditional? (hint: try using the comparison expression as the return value)